src/background/sync/googledrive.js

```diff
@@ -125,7 +125,7 @@ const GoogleDrive = BaseService.extend({
     if (url.startsWith(redirectUri)) {
       this.authState.set('authorizing');
       this.authorized({
-        code: url.split('#')[0].slice(redirectUri.length),
+        code: decodeURIComponent(url.split('#')[0].slice(redirectUri.length)),
```

Comment:
Is this safe for other browsers?

Reply:
I've verified it works in Chrome, yes.

Reply:
Actually I'm surprised the old code worked...

Reply:
Works in Firefox too.

Reply:
> Actually I'm surprised the old code worked... because I would have expected that URL parameters are always encoded.

That's true. 🤣